src/pages/NewChatPage.js

```diff
@@ -65,23 +80,39 @@ class NewChatPage extends Component {
             props.reports,
             props.personalDetails,
             props.betas,
+            '',
+            [],
+            this.props.groupChat ? EXCLUDED_GROUP_EMAILS : [],
         );
-
         this.state = {
             searchValue: '',
             recentReports,
             personalDetails,
+            selectedOptions: [],
             userToInvite,
         };
     }
 
     /**
      * Returns the sections needed for the OptionsSelector
      *
+     * @param {Boolean} maxParticipantsReached
      * @returns {Array}
      */
-    getSections() {
+    getSections(maxParticipantsReached) {
         const sections = [];
+        if (this.props.groupChat) {
+            sections.push({
+                title: undefined,
```

Comment:
On line `133`, should that also be `title: undefined`?

Reply:
Oops.